doc/rst/tools/c2chapel/c2chapel.rst

```diff
@@ -0,0 +1,125 @@
+.. _c2chapel:
+
+.. default-domain:: chpl
+
+---------
+c2chapel
+---------
+A tool to generate C bindings for Chapel
+-----------------------------------------------------
+
+``c2chapel`` is a tool to help the Chapel programmer generate extern types,
+records, and procedures when given a C99 file. For example, given the following
+function declaration in a header file:
+
+.. code-block:: c
+
+  void foo(char* str, int n);
+
+``c2chapel`` will generate Chapel code similar to:
+
+.. code-block:: chapel
+
+  extern proc foo(str: c_string, n : c_int) : void;
+
+
+Prerequisites
+-------------
+
+This tool has the following prerequisites:
+
+- A Unix-like environment (something with commands like cd, mkdir, rm, etc.)
+- A GNU-compatible version of 'make'
+- A C preprocessor
+- Python 2.7 or Python 3.5
+
+Other versions of Python may work, but only 2.7 and 3.5 have been tested as of
+version 0.1.0.
+
+Building
+--------
+
+``c2chapel`` is built on top of `pycparser <https://github.com/eliben/pycparser>`_.
+Building ``c2chapel`` will install pycparser into a local virtualenv, leaving
+the user's python environment untouched. A symbolic link will be created in the
+appropriate directory under ``$CHPL_HOME/bin``.
+
+To build, run ``make c2chapel`` from ``$CHPL_HOME``.
```

Comment:
This didn't work for me.
```
virtualenv /Users/diten/chapel/tools/c2chapel/install/venv
/bin/bash: virtualenv: command not found
```
Is there a prerequisite that a virtualenv be set up somehow?

Reply:
Yes, though I (apparently incorrectly) thought having python implied having virtualenv. I'll add that to the prerequisites.

Reply:
I'm able to build chpldoc and our test environment with `make chpldoc` or `make test-venv`, and they both use virtualenv.  Can the Makefile be set up like for them so it just works?

Reply:
Possibly, but I'm not sure what it would take to get that working. Could that be part of a later version of c2chapel?

Reply:
I think it would be really nice to use the same trick the other tools use so virtualenv isn't a prerequisite.  If it won't be straightforward then it can wait until later.